autograder/handgrading/tests/test_views/test_comment_views.py

```diff
@@ -104,11 +104,12 @@ def setUp(self):
             "text": "Sample comment text.",
         }
 
```

Comment:
In the comment tests, I would start out with "handgraders_can_leave_comments" as false, and then only set it to true in the tests where you're making sure that handgraders can leave comments when it's true.